packages/esm-patient-common-lib/src/orders/useOrders.ts

```diff
@@ -56,21 +56,12 @@ export function usePatientOrders(
   };
 }
 
-export function useOrderTypes() {
-  const orderTypesUrl = `${restBaseUrl}/ordertype`;
-  const { data, error, isLoading, isValidating } = useSWR<FetchResponse<OrderTypeFetchResponse>, Error>(
-    orderTypesUrl,
-    openmrsFetch,
-  );
-
-  return {
-    data: data?.data?.results ?? null,
-    error,
-    isLoading,
-    isValidating,
-  };
-}
-
 export function getDrugOrderByUuid(orderUuid: string) {
   return openmrsFetch(`${restBaseUrl}/order/${orderUuid}?v=${drugCustomRepresentation}`);
 }
+
+// See the Urgency enum in https://github.com/openmrs/openmrs-core/blob/492dcd35b85d48730bd19da48f6db146cc882c22/api/src/main/java/org/openmrs/Order.java
+export const priorityOptions: PriorityOption[] = [
+  { value: 'ROUTINE', label: 'Routine' },
+  { value: 'STAT', label: 'Stat' },
+];
```

Comment:
Are these anywhere close to being translatable? It seems like the only thing we should be storing here are the `values` and translating them in the components where they are used?